tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/AdoNet/MicrosoftDataSqlClientTests.cs

```diff
@@ -59,6 +59,7 @@ public void SubmitsTraces(string packageVersion)
             const string expectedOperationName = dbType + ".query";
             const string expectedServiceName = "Samples.Microsoft.Data.SqlClient-" + dbType;
 
+            using var telemetry = this.ConfigureTelemetry();
```

Comment:
You should also add telemetry to `FakeCommandTests.cs` which should assert `IntegrationId.AdoNet`

Reply:
Done in https://github.com/DataDog/dd-trace-dotnet/pull/2241/commits/d3fe9d7c26b0736576427a7b8d025d567dfeb32f 🤞